save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/info/UserInfo.kt

```diff
@@ -19,10 +21,17 @@ import kotlinx.serialization.Serializable
  * @property twitter
  * @property organizations
  * @property globalRole
+ * @property id
+ * @property isActive
+ * @property oldName
+ * @property originalLogins
  */
 @Serializable
 data class UserInfo(
     val name: String,
+    val id: Long? = null,
+    val oldName: String? = null,
```

Comment:
What is `oldName`? Is it the previous login?

Reply:
Yes. This is necessary to change the name. This is the new name and the previous one.

Reply:
Why do we store only the previous name then? User may change the name multiple times.

Reply:
We need the old name only in the wrapper for the front, and only in order to find the user in the database by it.

Reply:
Ah, OK, so this field should be `null` except for the renaming process. Could you please add this to class KDoc?

Reply:
Added